server/api/controllers/auth/post-signup.js

```diff
@@ -77,13 +84,15 @@ the account verification message.)`,
     emailAddress,
     password,
     fullName,
+    nationality,
     lifestage,
     phoneNumber,
   }) {
     if (
       emailAddress == null ||
```

Comment:
probs have to check for empty strings too

Reply:
nationality = null and nationality = "" is different?

Reply:
yep, `nationality == null` only checks for null and undefined